### DISCUSSION TITLE
reentrancy guard

### DISCUSSION THREAD

**Discussion by frankomosh** [Cyfrin/security-and-auditing-full-course-s23#110] [General]
Hello everyone.. can a reentrancy guard protect an external function from an inflation attack?

**Comment by tobezhanabi** [comment]
An inflation attack, typically involves manipulating the total supply of a cryptocurrency or token. This is often achieved by exploiting vulnerabilities in the token contract that allow unauthorized minting of new tokens. so NO

**Reply by frankomosh** [reply]
okey
